src/main/java/io/confluent/examples/streams/microservices/InventoryService.java

```diff
@@ -45,7 +45,7 @@
 public class InventoryService implements Service {
 
   private static final Logger log = LoggerFactory.getLogger(InventoryService.class);
-  public static final String INVENTORY_SERVICE_APP_ID = "inventory-service";
+  public static final String SERVICE_APP_ID = "InventoryService";
```

Comment:
`getClass().getSimpleName();` and remove `static` to align with code above?

Reply:
Hmmm... Not sure atm -- the only difference to the other classes is `private` vs `public` ?

Reply:
None of the other tests (except for `InventoryServiceTest` reference the SERVICE_APP_ID, which is why this is the only one that is different.